src/blueprint/azext_blueprint/_help.py

```diff
@@ -0,0 +1,451 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['blueprint'] = """
+    type: group
+    short-summary: Commands to manage blueprint.
+"""
+
+helps['blueprint create'] = """
+    type: command
+    short-summary: Create a blueprint definition.
+    examples:
+      - name: SubscriptionBlueprint
+        text: |-
+               az blueprint create --subscription "00000000-0000-0000-0000-000000000000" --name \\
+               "simpleBlueprint" --description \\
+               "blueprint contains all artifact" --target-scope \\
+               "subscription"
+      - name: ManagementGroupBlueprint
+        text: |-
+               az blueprint create --management-group myManagementGroup --name \\
+               "simpleBlueprint" --description \\
+               "blueprint contains all artifact" --target-scope \\

```

Comment:
Do you have a default value for target scope?

Reply:
Yes, default is `subscription`

Reply:
The same as what I have mentioned in https://github.com/Azure/azure-cli-extensions/pull/1164/files#r392013317.
should the management group be in the same tenant with login account subscription?